add-on/TA-Demisto/README.md

```diff
@@ -21,16 +19,57 @@ Supporting Add-on for Demisto. This application allows a user to create incident
 * This app can be installed through UI using "Manage Apps" or from the command line using the following command:
 $SPLUNK_HOME/bin/splunk install app $PATH_TO_SPL/TA-Demisto.spl/
 
+# HTTPS Certificate Validation
+
+The add-on uses secure communication with Demisto.
+
+On few occasions it would be necessary to manually set a certificate for proper verification.
```

Comment:
Should mention that users that had previous versions of the app and disabled certificate will have to do it again.